omegatrade-app/backend/user.js

```diff
@@ -0,0 +1,29 @@
+'user strict';
+var database = require('./app/config/database.js');
+var Users = function () { }
+Users.prototype.getAllUsers = async function () {
+    try {
+        const query = {
+            sql: 'select * from company',
+        };
```

Comment:
You can instruct the Spanner client to automatically transform the result to JSON like this:

```suggestion
        const query = {
            sql: 'select * from company',
            json: true,
        };
```